java/src/main/java/org/wildfly/openssl/Identification.java

```diff
@@ -30,6 +32,7 @@
     static final String CPU_ID;
     static final String ARCH_NAME;
     static final String[] NATIVE_SEARCH_PATHS;
+    private static final Pattern RHEL_PATTERN = Pattern.compile(".*\\.(el[678])\\..*");
```

Comment:
Maybe too far in the future, but should this be `.*\\.(el[0-9]+)\\..*`?

Reply:
The matched value (i.e., `el6`, `el7`, or `el8`) will be used to determine the module from which the native library should be loaded from (e.g., `el8-x86_64`). This module needs to actually exist in order to be used successfully. Since we currently only provide `el6-{i386,x86_64}`, `el7-x86_64`, and `el8_x86_64` modules, I think we could wait to update this pattern to support a future version of RHEL until the RHEL version is actually available. At that point, in addition to updating this pattern, we'd also add a corresponding new module to the build.